### PR TITLE
Less strict matching for beta versions (bug 1143608)

### PR DESCRIPTION
**Pull request by wagnerand**

### CODE REVIEW

File: apps/constants/base.py [hunk 1]
```diff
@@ -354,7 +354,10 @@
 # Contributions.
 FOUNDATION_ORG = 1  # The charities.id of the Mozilla Foundation.
 
-VERSION_BETA = re.compile('(a|alpha|b|beta|pre|rc)\d*$')
+VERSION_BETA = re.compile(r"""(a|alpha|b|beta|pre|rc) # Either of these
```
Comment (eviljeff) [inline apps/constants/base.py]: Why the change to a multi-line expression? It fits on 1 line and the comment could be on the line above.

Comment (magopian) [inline apps/constants/base.py]: Looks good to me ;)

**Comment by magopian** [comment]
Thanks a lot @wagnerand ;)
r+
